intellst-api/config/routes.yaml

```diff
@@ -1,3 +1,4 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+
```

Comment:
extra line

Reply:
![Screenshot from 2020-06-20 10-27-55](https://user-images.githubusercontent.com/45236851/85196142-bb1ed580-b2e0-11ea-9eaa-bd23c6c242b4.png)
I don't understand localy is totaly diferrent this file